### DISCUSSION TITLE
Error when trying to run samples

### DISCUSSION THREAD

**Discussion by cmousikos** [KhronosGroup/Vulkan-Samples#615] [Q&A] [answered]
Hi, i am experiencing a problem when i try to run samples. The error i get is in the assets. It says that it fails to open file assets/fonts/Roboto-Regular.ttf. I tried include_directories("./assets")  at the CMakeLists.txt of the vulkan-sample project, but still the error occurs. Here is a printscreen of the error i get. Do you have any idea of how i can overcome this? 

![Untitled](https://user-images.githubusercontent.com/74675716/217569252-f60d071e-6a25-49bd-8a08-f4040c4d7951.png)

**Comment by SaschaWillems** [comment]
Did you clone the repo with submodules as noted under https://github.com/KhronosGroup/Vulkan-Samples#setup?

**Reply by cmousikos** [reply]
Yes of course, and i can see the assets folder. I followed the instructions, carefully, step by step.
![Untitled](https://user-images.githubusercontent.com/74675716/217572022-8e98d314-a781-4e77-b002-eee0831aa555.png)

**Reply by gpx1000** [reply]
With submodules, you could see a blank folder if git doesn't correctly pull down the entire submodule.  To test that, make certain you can see assets/fonts/Roboto-Regular.ttf  If you see a blank folder, then you need to ensure that the submodules are getting updated.  Something similar to `git submodule --init --recursive update` should do it.  However, that knowledge is documented https://github.com/KhronosGroup/Vulkan-Samples#setup Which might be the better way of doing it.

**Reply by cmousikos** [reply]
I can see them, i executed the following command : git clone --recurse-submodules https://github.com/KhronosGroup/Vulkan-Samples.git
![image](https://user-images.githubusercontent.com/74675716/217576920-c2e489e8-55dd-4a81-8256-598d94354c28.png)

**Comment by gpx1000** [comment, marked as answer]
Then the next thing that it could be.  Do you notice how the console error line has a relative path?  (i.e. it starts with assets/fonts).  Are you running the program from the top level directory?  If it's getting built to a subdirectory than that path isn't correct.  Basically you need the working directory to be Vulkan-Samples, so if you run it from there then those paths should resolve.

**Reply by cmousikos** [reply]
Thank you, it worked! i was running the vulkan_samples from Vulkan-Samples\build\windows\app\bin\Release\AMD64
